lib/lev/background_job.rb

```diff
@@ -20,7 +20,7 @@ class BackgroundJob
       STATE_UNKNOWN
     ].freeze
 
-    def initialize(attrs = {})
+    def initialize(attrs = {}, write = true)
```

Comment:
can we make this an options hash so that when callers use it, it looks like `write: true` instead of `true` without knowing what the true means?